pcmanfm/desktopwindow.cpp

```diff
@@ -948,7 +948,7 @@ bool DesktopWindow::eventFilter(QObject* watched, QEvent* event) {
             break;
         }
     }
-    return false;
+    return Fm::FolderView::eventFilter(watched, event);
```

Comment:
No, you **should not** add new override -> implement `View::eventFilter()`. You need to

> just change a line to fix a bug

in this way
```c++
return View::eventFilter(watched, event);
```
to protect against (a potential) new bug in the future.

> I feel that you make no hierarchy of things according to their relative importance in your mind.

Of course I do. Everyone does. But I always prefer to make a "clean" (for the future) fix instead of  an "it works somehow" fix. With this I'm not saying this patch is one of "it works somehow".

I just tried to point that the solution can be made more robust for the future. That's all.